Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,26 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+Add your answers to the Algorithms exercises here.
+
+Exercise 1
+a) O(n)
+b) O(n^n)
```

Comment:
O(n^3): There are 4 nested for loops. All of them are going to run in linear time, except for the last one, which will always run 10 times (constant time, which can be dropped).